cloud/terraform/aws/backup/modules/backup-plan/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_backup_plan" "this" {
```

Comment:
For resources where there is only one, it's convention to pick a term like `main` or `this`: https://cloud.google.com/docs/terraform/best-practices/general-style-structure#naming-convention,
https://developer.hashicorp.com/terraform/plugin/best-practices/naming#attribute-names
that way we don't have to make pointless decisions/arguments about naming each time we create a resource that is the only one of its kind in the module.

Reply:
Discussed via call. Will go with `this`.